examples/CMakeLists.txt

```diff
@@ -76,7 +79,8 @@ if(FORCE_EXAMPLES OR EXISTS ${CMAKE_INSTALL_PREFIX}/share/dlite/examples)
     test_example(ex4)
   endif()
   #test_example(ex5d)
-
+  if(WITH_PYTHON)
+  endif()
```

Comment:
Shouldn't this be on line 105, after the foreach-loop?

Reply:
Line 82-83 should simply be removed... well spotted